docs/ProjectProperties.md

```diff
@@ -0,0 +1,67 @@
+# Project Properties
+
+[<< Back to index](./index.md)
+
+FirstSpirit provides a mechanism to configure _global project settings_ in a specific area of the project data. In the **FirstSpirit SiteArchitect** this **Global settings** area is presented as the 'gray' section found behind the cog wheels.
+
+![Global setting](./imgs/GlobalContentAreaSA.png)
+
+In the FSXA example project [Smart Living Global](https://docs.e-spirit.com/module/fsxa/developer-dokum/fsxa-javascript/components/smart_living_global_1) a "Theming" tab allows you to quickly modify *project settings* such as button color, *text highlighting color* or *banner opacity*.
+
+### Reading a single value
+
+When developing a PWA the _global settings_ can be a accessed from any FSXABaseCompenent element via the _globalSettings()_ method. This way reading the text highlight color value from the Smart Living Global project is as simple as doing:
+
+```javascript
+import { FSXABaseAppLayout } from 'fsxa-pattern-library'
+class AppLayout extends FSXABaseAppLayout {
+  render() {
+    <div>Color: {this.globalSettings?.data.ps_text_highlighting}</div>
+    )
+  }
+}
+export default AppLayout
+```
+
+The key _ps_text_highlighting_ is the input field name defined in *Smart Living Global*'s project settings template (UID: *project_settings*):
+
+```xml
+<CMS_INPUT_TEXT name="ps_text_highlighting" hFill="yes" singleLine="no" useLanguages="no">
+    <LANGINFOS>
+        <LANGINFO lang="*" label="Text-Highlighting color"/>
+        <LANGINFO lang="DE" label="Text-Highlighing Farbe"/>
+    </LANGINFOS>
+</CMS_INPUT_TEXT>
+```
+
+### Reading a list of values
+
+In *Smart Living Global* you see two links in each page's bottom right corner:
```

Comment:
"In the footer" instead of "in each page's right corner" would be more precise in my opinion.